web-common/src/main/java/edu/unc/lib/boxc/web/common/utils/SerializationUtil.java

```diff
@@ -140,6 +140,10 @@ public static Map<String, Object> metadataToMap(ContentObjectRecord metadata, Ac
             result.put("type", metadata.getResourceType());
         }
 
+        if (metadata.getCollectionId() != null) {
+            result.put("collectionId", metadata.getCollectionId());
```

Comment:
we're only doing it with some of the attributes at the moment, but generally it would be good to use the constant for these field names, so here the key could be `SearchFieldKey.COLLECTION_ID.getUrlParam()`.